src/main/java/cms/gov/madie/measure/services/TestCaseService.java

```diff
@@ -765,17 +769,48 @@ protected void defaultTestCaseJsonForQdmMeasure(TestCase testCase, Measure measu
   }
 
   protected void checkTestCaseSpecialCharacters(TestCase testCase) {
-    if (StringUtils.isBlank(testCase.getTitle()) || StringUtils.isBlank(testCase.getSeries())) {
-      throw new InvalidRequestException("Test Case title and group are required");
+    if (StringUtils.isBlank(testCase.getTitle())) {
+      throw new InvalidRequestException("Test Case title is required.");
     }
     Pattern special = Pattern.compile("[(){}\\[\\]<>/|\"':;,.~`!@#$%^&*_+=\\\\]");
     Matcher titleHasSpecial = special.matcher(testCase.getTitle());
-    Matcher groupHasSpecial = special.matcher(testCase.getSeries());
     if (titleHasSpecial.find()) {
       throw new SpecialCharacterException("Title");
     }
-    if (groupHasSpecial.find()) {
-      throw new SpecialCharacterException("Group");
+    if (StringUtils.isNotBlank(testCase.getSeries())) {
+      Matcher groupHasSpecial = special.matcher(testCase.getSeries());
+      if (groupHasSpecial.find()) {
+        throw new SpecialCharacterException("Group");
+      }
+    }
+  }
+
+  protected TestCaseImportOutcome checkErrorSpecialChar(
+      String model, TestCaseImportRequest testCaseImportRequest) {
+    if (ModelType.QDM_5_6.getValue().equalsIgnoreCase(model)) {
+      try {
+        checkTestCaseSpecialCharacters(
+            TestCase.builder()
+                .title(
+                    testCaseImportRequest.getGivenNames() != null
+                        ? testCaseImportRequest.getGivenNames().get(0)
+                        : null)
+                .series(testCaseImportRequest.getFamilyName())
+                .build());
+      } catch (InvalidRequestException ex) {
+        return TestCaseImportOutcome.builder()
+            .patientId(testCaseImportRequest.getPatientId())
+            .successful(false)
+            .message("Test Case Title is required.")
+            .build();
+      } catch (SpecialCharacterException ex) {
+        return TestCaseImportOutcome.builder()
+            .patientId(testCaseImportRequest.getPatientId())
+            .successful(false)
+            .message("Test Cases Group or Title cannot contain special characters.")
```

Comment:
```suggestion
            .message(ex.getMessage())
```

Reply:
@adongare Thanks Ashok! Adopted the change.

Reply:
@adongare Fixed the test too. Thanks!

Reply:
thanks Cecilia! I think this is another similar case or maybe forgot to push the change?

Reply:
@adongare No for this exception, the message has to be Test Cases Group or Title cannot contain special characters. (based on the MAT-6992 requirement). The ex.getMessage will give: Test Case Group (or Title) can not have special characters followed by the list of the special characters, so they are not the same.